src/display.rs

```diff
@@ -418,8 +422,8 @@ impl Display {
         let Point{line: Line(row), col: Column(col)} = terminal.cursor().point;
         let SizeInfo{cell_width: cw,
                     cell_height: ch,
-                    padding_x: px,
-                    padding_y: py, ..} = *terminal.size_info();
+                    padding_left: px,
+                    padding_bottom: py, ..} = *terminal.size_info();
```

Comment:
not really sure about this part. I don't know what this does, but I assume it's a coordinate starting from the left.